backend/app/services/share/share_webhook.py

```
@@ -0,0 +1,284 @@
+                f"User {applicant_name} ({email_display}) "
+                f"requested {permission_level} permission for {resource_type_name} '{resource_name}'"
+            ),
+            status="pending",
+            detail_url=action_url,
+        )
+
+        # Send notification synchronously (we're already in a background task)
+        return webhook_notification_service.send_notification_sync(notification)
+
+    except Exception as e:
+        logger.exception(f"Error sending share request notification: {e}")
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)